templates/commands/render/render_test.go

```diff
@@ -719,26 +708,33 @@ steps:
 				t.Errorf("dest directory contents were not as expected (-got,+want): %s", diff)
 			}
 
-			gotBackupContents := common.LoadDirWithoutMode(t, backupDir)
-			gotBackupContents = stripFirstPathElem(gotBackupContents)
+			var gotBackupContents map[string]string
+			backupSubdir, ok := mustGlob(t, filepath.Join(backupDir, "*")) // When a backup directory is created, an unpredictable timestamp is added, hence the "*"
+			if ok {
+				gotBackupContents = common.LoadDirWithoutMode(t, backupSubdir)
+			}
 			if diff := cmp.Diff(gotBackupContents, tc.wantBackupContents, cmpopts.EquateEmpty()); diff != "" {
 				t.Errorf("backups directory contents were not as expected (-got,+want): %s", diff)
 			}
 		})
 	}
 }
 
-// Since os.MkdirTemp adds an extra random token, we strip it back out to get
-// determistic results. Input map keys should use forward slash separator.
-func stripFirstPathElem(m map[string]string) map[string]string {
-	out := map[string]string{}
-	for k, v := range m {
-		// Panic in the case where k has no slashes; this is just a test helper.
-		elems := strings.Split(k, "/")
-		newKey := path.Join(elems[1:]...)
-		out[newKey] = v
+func mustGlob(t *testing.T, glob string) (string, bool) {
```

Comment:
https://github.com/abcxyz/readability/wiki/Go-Best-Practices#test-helper We often prefix test helpers with "test" prefix, e.g. `testMustGlob`, for the purpose of identifying test helpers more easily

Reply:
Done.